src/dnx/servicing.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "version.h"
+#include "utils.h"
+#include <string>
+#include <fstream>
+#include <algorithm>
+#include "TraceWriter.h"
+
+namespace
+{
+    const std::string runtime_moniker =
+#if defined(CORECLR_WIN)
+        "coreclr"
+#else
+        "clr"
+#endif
+        "-win-"
+#if defined(AMD64)
+        "x64"
+#elif defined(ARM)
+        "arm"
+#else
+        "x86"
```

Comment:
Will think about that. It would have to be separate though since we are building a string literal here.